src/main/java/frc/robot/commands/AlignToPose.java

```diff
@@ -20,6 +22,15 @@ public class AlignToPose extends Command {
   private PIDController PID_Y;
   private PIDController PID_Rotation;
 
+  private final double ELEVATOR_UP_SLEW_RATE = 0.5;
+
+  private final SlewRateLimiter angularVelocityLimiter = new SlewRateLimiter(ELEVATOR_UP_SLEW_RATE);
```

Comment:
The angular velocity will need a different constant, since it's an angle, not a coordinate